website/docs/gator.md

```diff
@@ -0,0 +1,180 @@
+---
+id: gator
+title: The gator CLI
+---
+
+The `gator` CLI is a tool for testing Gatekeeper ConstraintTemplates and
```

Comment:
I'm actually about to write a doc proposing another `gator` feature (`validate`).  Could we do something more general?

The `gator` CLI is a tool for evaluating Gatekeeper ConstraintTemplates and Constraints in a local environment.

Reply:
1. Yay!
2. Will do.